src/pool/srv.c

```diff
@@ -41,11 +40,6 @@ init(void)
 	if (rc)
 		D_GOTO(err_pool_iv, rc);
 
-	ec_agg_disabled = false;
-	d_getenv_bool("DAOS_EC_AGG_DISABLE", &ec_agg_disabled);
-	if (unlikely(ec_agg_disabled))
-		D_WARN("EC aggregation is disabled.\n");
```

Comment:
oh, because disable this might cause serious issue for the test, i.e. stable epoch will never proceed, which will cause serious consequence.  Just to avoid the trouble here, so disable this option anyway.